Client/Frontend/Library/RecentlyClosedTabsPanel.swift

```diff
@@ -126,6 +129,15 @@ class RecentlyClosedTabsPanelSiteTableViewController: SiteTableViewController {
         return self.recentlyClosedTabs.count
     }
 
+    // MARK: - Libray Toolbar actions
+    func handleBackButton() {
+        // no implementation needed
+    }
+
+    func handleDoneButton() {
+        // no implementation needed
+    }
```

Comment:
I wish we could mark functions or variables in protocols as optional - as in, implement this, but, if you don't and someone calls it, do nothing. Just some syntactical sugar of sorts.